nailgun/entities.py

```diff
@@ -1487,7 +1506,7 @@ def __init__(self, server_config=None, **kwargs):
                 OperatingSystem,
                 null=True,
             ),
-            'parent': entity_fields.OneToOneField(HostGroup, null=True),
+            'ancestry': entity_fields.StringField(null=True),
```

Comment:
I think it will not be so obvious for people if we use field name that does not have any correspondence to real attribute.

And when I said 'hash', I mean your comment to the code:
        # The server returns OneToOneFields as a hash of attributes, like so:
        #
        #     {'name': 'Alice Hayworth', 'login': 'ahayworth', 'id': 1}
        #
        # OneToManyFields are a list of hashes.

And actually it usually does that in mentioned way, but okay, I can follow your comment and implement functionality in that way